test/test-mda-models/aissemble-test-data-delivery-pyspark-model-basic/pyproject.toml

```diff
@@ -1,4 +1,5 @@
 # GENERATED STUB - PLEASE ***DO*** MODIFY
+# Generated from: templates/data-delivery-pyspark/pyproject.toml.vm
```

Comment:
A:
```suggestion
# Generated from: ${templateName}
```

Reply:
Since this is the test module, wouldn't the path be the generated component and not the VelocityContext's variable?